draft-ietf-tls-tls13.md

```diff
@@ -5183,6 +5183,47 @@ preference to crafting a new one. Many adequate cryptographic libraries
 are already available under favorable license terms.  Should those prove
 unsatisfactory, {{RFC4086}} provides guidance on the generation of random values.
 
+TLS uses random values in various places, some of which are made public (e.g.
+ClientHello.random) and others of which are kept private and used as input to
+crytpographic functions (e.g. Diffie-Hellman private values).
+
+There have been well-documented attacks [checkoway] where bad actors have
```

Comment:
You need to create a citation here or this will fail.  `{{?CHECKOWAY=DOI.10.1145/2976749.2978395}}` should do the trick.